robolectric/src/test/java/org/robolectric/shadows/ShadowAudioManagerTest.java

```diff
@@ -1538,6 +1541,24 @@ public void getDirectProfilesForAttributes_returnsEmptyListByDefault() {
     verify(callback).onAudioDevicesAdded(new AudioDeviceInfo[] {outputDevice});
   }
 
+  @Test
+  public void audioManager_activityContextEnabled() {
+    String originalProperty = System.getProperty("robolectric.createActivityContexts", "");
+    System.setProperty("robolectric.createActivityContexts", "true");
+    try {
+      AudioManager applicationAudioManager = appContext.getSystemService(AudioManager.class);
+      Activity activity = Robolectric.setupActivity(Activity.class);
+      AudioManager activityAudioManager = activity.getSystemService(AudioManager.class);
+      assertThat(applicationAudioManager).isNotSameInstanceAs(activityAudioManager);
```

Comment:
I think adding a ctesque test here is out of scope for this PR. The goal is to migrate it to use static state. Also, Robolectric does not run any real system services, and I am not sure to what extent AudioManager works in emulators.

If it is easy to add a ctesque test, then it would be okay, but I would rather not spend too much time on that.

Reply:
Ah okay that makes sense